lib/screens/v2/widget_instance/normal_header.ex

```diff
@@ -2,23 +2,33 @@ defmodule Screens.V2.WidgetInstance.NormalHeader do
   @moduledoc false
 
   alias Screens.V2.WidgetInstance.NormalHeader
+  alias Screens.Config.{Screen, Solari}
 
   defstruct screen: nil,
-            station_name: nil,
+            icon: nil,
+            text: nil,
             time: nil
 
-  @type config :: :ok
+  @type icon :: :logo | :x | :green_b | :green_c | :green_d | :green_e
   @type t :: %__MODULE__{
-          screen: config(),
-          station_name: String.t(),
+          screen: Screens.Config.Screen.t(),
+          icon: icon | nil,
+          text: String.t(),
           time: DateTime.t()
         }
 
   defimpl Screens.V2.WidgetInstance do
     def priority(_instance), do: [2]
 
-    def serialize(%NormalHeader{station_name: station_name, time: time}) do
-      %{station_name: station_name, time: DateTime.to_iso8601(time)}
+    def serialize(%NormalHeader{icon: icon, text: text, time: time}) do
```

Comment:
So I guess I could do something more concise like this:
```
def serialize(%{time: t} = header) do
  %{t | time: DateTime.to_iso8601(t)}
end
```
...but it's kind of nice to be explicit about what `serialize` is returning anyway, so I don't feel strongly that this is better. (Do you?)

Reply:
Oh sorry, I just meant you could do this:
```suggestion
    def serialize(%{icon: icon, text: text, time: time}) do
```

I think `%{t | time: DateTime.to_iso8601(t)}` would actually not work because the hidden `__struct__` field would stick around, keeping the return value as a struct which `Jason` wouldn't know what to do with.